addons/l10n_ke_edi_tremol/models/account_move.py

```diff
@@ -82,6 +94,12 @@ def _l10n_ke_validate_move(self):
 
     def _l10n_ke_fiscal_device_details_filled(self):
         self.ensure_one()
+        # If the company is configured for OSCU, don't block the Send & Print.
+        if (
+            'l10n_ke_oscu_is_active' in self.company_id
+            and self.company_id.l10n_ke_oscu_is_active
+        ):
```

Comment:
```suggestion
        if  self.company_id.l10n_ke_oscu_is_active:
```

Is it not enough like this ? :thinking: